lnwallet/chainfee/estimator_test.go

```diff
@@ -315,3 +326,41 @@ func TestGetCachedFee(t *testing.T) {
 		})
 	}
 }
+
+func TestRandomFeeUpdateTimeout(t *testing.T) {
+	t.Parallel()
+
+	var (
+		minFeeUpdateTimeout = 1 * time.Minute
+		maxFeeUpdateTimeout = 2 * time.Minute
+	)
+
+	estimator, _ := NewWebAPIEstimator(
+		nil, false, minFeeUpdateTimeout, maxFeeUpdateTimeout,
+	)
+
+	for i := 0; i < 1000; i++ {
+		timeout := estimator.randomFeeUpdateTimeout()
+		require.True(t, timeout >= minFeeUpdateTimeout, "timeout %v "+
```

Comment:
I think we need `require.Truef` to properly print the error msg.

Reply:
Fixed in updated squashed commit.